content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -263,8 +263,10 @@ The following table lists the supported configuration variables that are availab
 | `DD_TRACE_DEBUG`<br/><br/>`DebugEnabled`           | Enables or disables debug logging. Valid values are: `true` or `false` (default).                                                                                                                                 |
 | `DD_TRACE_HEADER_TAGS`<br/><br/> `HeaderTags`       | Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on root spans. (e.g. : `CASE-insensitive-Header:my-tag-name,User-ID:userId`). Added in version 1.18.3      |
 | `DD_TAGS`<br/><br/>`GlobalTags`                     | If specified, adds all of the specified tags to all generated spans (e.g., `layer:api,team:intake`). Added in version 1.17.0                                  |
+| `DD_TRACE_LOGGING_RATE`                             | Sets rate limiting for log messages. If set, unique log lines will be written once per `x` seconds. For example, to log a given message once per 60s, set to `60`. Setting to `0` disables log rate limiting. Disabled by default |
```

Comment:
In what version was this setting added?

Reply:
done in https://github.com/DataDog/documentation/pull/10221/commits/51e9f8bdf7ab18c1f41b3d6b64bb496b7755c008